sBTC SIP

```diff
@@ -0,0 +1,166 @@
+    - The deposit transaction contains a UTXO (deposit UTXO) spendable by sBTC Signers, with an OP_DROP payload.
+    - The payload contains the recipient address of the sBTC among other relevant info for the deposit.
+    - The relevant info could contain fee suggestion or max_fee.
+2. **Proof of deposit:** The bitcoin holder submits a proof of deposit on Stacks by invoking the Signer API.
+3. **Deposit accept:**
+4. **Deposit redeem:** The sBTC Signers redeem the deposit by consuming the deposit UTXO, consolidating it into the sBTC UTXO.
+5. **Mint:** The sBTC Signers finalize the deposit acceptance by calling the mint function in .sbtc to mint the sBTC.
+
+### Withdrawals (Redeeming sBTC)
+A clarity call initiates the withdrawal process that successfully completed after 6 sortitions paying to the specified Bitcoin address. Failure cases can be resolved earlier than 6 sortitions, but some will only fail after the sBTC Bootstrap Signer attempts to create the withdrawal transaction.
+
+**Six Sortitions for Withdrawal:** The sBTC Bootstrap signers will only trigger the withdrawal on Bitcoin after the Stacks withdrawal has been confirmed by 6 sortitions. Bitcoin transactions cannot be constructed such that they’re valid for only a specific Bitcoin fork, nor can they be forcefully taken back after they’re created, and a Bitcoin fork can cause a Stacks reorg such that the initial withdrawal request becomes invalid. The sBTC Bootstrap Signers will need to wait for the withdrawals to be sufficiently final in order to make the withdrawal on Bitcoin without risk of allowing a double withdrawal from the Stacks Blockchain. The withdrawal procedure will need to track the number of valid sortitions after the withdrawal to verify that the Stacks transaction is final, and 6 is the standard choice for Bitcoin finality.
+
+#### Withdrawal Flow
+The main steps of the sBTC withdrawal flow are as follows:
+1. **Withdrawal request:** An sBTC holder calls the withdraw-request function in the .sbtc contract.
+    - This transfers the requested amount of sBTC to the .sbtc contract & mints the user a non-transferable locked-sBTC as a placeholder.
+2. **Withdrawal accept:** If accepted, the following happens:
+    - The signers create a "Withdrawal accept" transaction on Bitcoin which returns the requested amount to the designated address.
+    - Once the Bitcoin transaction is confirmed, the signers must then call into the sBTC protocol to mark it as complete - this is done through a Stacks transaction.
+    - If successful, the resulting Stacks transaction will record the withdrawal request as complete & will accordingly burn the user’s locked-sBTC.
+3. **Withdrawal reject:** If instead the request is rejected, the sBTC signers will call the withdraw-reject function in the .sbtc smart contract. This function does the following:
+    - Returns the sBTC to the holder.
+    - Records the signer votes.
```

Comment:
This describes the happy path.  What about the unhappy paths?  Things to consider:

* What happens if signers can't reach consensus to fulfill the sBTC withdrawal?
* How do signers mark the sBTC withdrawal as complete?  Are we trusting a threshold of them to be honest, or do they also have to provide a proof of some kind that the actually completed the withdrawal (e.g. they could post the BTC transaction itself to the contract, and the contract could validate it)
* What happens if the BTC transaction is never confirmed?  e.g. it has a low fee
* Who pays for the BTC transaction fee?
* Can there be more than one unconfirmed withdrawal at a time?  How does that affect the set of available UTXOs for the signers to use to issue new withdrawal transactions?
* What happens of the BTC transaction is double-spent?  For example, signers may need to RBF a transaction, or may even deliberately double-spend it to get it "un-stuck" in the event of multiple parallel withdrawals
* What happens of the Stacks transaction is never confirmed?  i.e. perhaps the fee is too low, or it gets RBF'ed?
* Who pays for the Stacks transaction fee?

Reply:
Described here: https://github.com/stacks-network/sbtc/issues/16

Reply:
I'm linking these for reference when we go back through and add details, not to say you should have looked here earlier.

Reply:
Thank you for that.  That issue thread answers some of the questions (and should be folded into the SIP), but I didn't see anything speaking to liveness issues that can arise if signers can't pay STX or BTC fees (or if their transactions get stuck)